hotel/report/hotel_report.py

```diff
@@ -0,0 +1,68 @@
+# See LICENSE file for full copyright and licensing details.
+
+import time
+from datetime import datetime
+
+from dateutil import parser
+from dateutil.relativedelta import relativedelta
+
+from odoo import api, fields, models
+from odoo.tools import DEFAULT_SERVER_DATETIME_FORMAT
+
+
+class FolioReport(models.AbstractModel):
+    _name = "report.hotel.report_hotel_folio"
+    _description = "Auxiliar to get the report"
+
+    def _get_folio_data(self, date_start, date_end):
```

Comment:
In v12+, I think it's better to use date types directly rather than string types.

Reply:
Done